app/ui/src/main/java/com/fsck/k9/activity/MessageList.java

```diff
@@ -946,21 +940,6 @@ public boolean onOptionsItemSelected(MenuItem item) {
         } else if (id == R.id.delete) {
             messageViewFragment.onDelete();
             return true;
-        } else if (id == R.id.reply) {
```

Comment:
Removing this disables the functionality in the "Send…" submenu of the action bar.